tests/open-ce-tests.yaml

```diff
@@ -0,0 +1,18 @@
+tests:
+   - name: SetupTensorFlow Models repo
+     command: |
+         echo "setup"
+         git clone --recursive -b v2.4.0 $(python -c "import tensorflow; print(tensorflow.__version__)")
```

Comment:
This clone statement looks like it has some typos. Should remove the hard coded version and add the git repo name.

Reply:
yeah - fixed. I had it hardcoded because TF was reporting the version as 2.4.0-rc4 and that doesn't match with the models repo. Matches now.